lib/twingly/livefeed/version.rb

```diff
@@ -0,0 +1,5 @@
+module Twingly
+  module LiveFeed
+    VERSION = "5.0.1"
```

Comment:
> But we could have this repo consist of two gems that we publish, one for search and one for livefeed.

Yes, not sure what's "best" though. 

BTW, I don't think we should release this as a gem, as a reference implementation this serves it's purpose in master, then we can iterate some more on it before releasing it. I think it would be best if we had Search v3 implemented as well before doing a release (no matter if it's one or two gems).